tests/python_functional/src/helpers/snmptrapd/snmptrapd.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
```

Comment:
This file contains a fixture definition for our testcases. In this case filename should be called as: "conftest.py"

Reply:
done

Reply:
thanks